SU2_CFD/src/solvers/CDiscAdjFEASolver.cpp

```diff
@@ -557,7 +435,38 @@ void CDiscAdjFEASolver::SetSensitivity(CGeometry *geometry, CConfig *config, CSo
         AD::ResetInput(Coord[iDim]);
       }
 
-      nodes->SetSensitivity(iPoint, iDim, Sensitivity);
+      if (!config->GetTime_Domain()) {
+        nodes->SetSensitivity(iPoint, iDim, Sensitivity);
+      } else {
+        nodes->SetSensitivity(iPoint, iDim, nodes->GetSensitivity(iPoint, iDim) + Sensitivity);
+      }
+    }
+  }
+
+  // Temporary Output
+  if (config->GetAdvanced_FEAElementBased()) {
+    if (rank == MASTER_NODE) {
+
+      /*--- Header of the temporary dv sensitivity output file ---*/
+      ofstream myfile2_res;
+
+      switch (config->GetDV_FEA()) {
+        case YOUNG_MODULUS: case DENSITY_VAL:
+          myfile2_res.open("grad_dv.opt");
+          myfile2_res << "INDEX"
```

Comment:
We need a better place for this.
There is another "temporary" section like this in CDiscAdjFEAIteration::Postprocess maybe you can simply call the various postprocessing in the EvaluateSensitivies method of the discrete adjoint multizone driver.